app.rb

```diff
@@ -30,22 +32,48 @@ def auth_server_uri
 
```

Comment:
Can you just do `@logout_msg = session.delete(:logout)` here rather than repeating yourself in the erb calls below?